server/segment_rpc.go

```diff
@@ -634,6 +635,15 @@ func genPayment(sess *BroadcastSession, numTickets int) (string, error) {
 		}
 
 		protoPayment.TicketSenderParams = senderParams
+
+		ratPrice, _ := ratPriceInfo(protoPayment.ExpectedPrice)
+		glog.V(common.VERBOSE).Infof("Created new payment - recipient=%v  faceValue=%v  winProb=%v  price=%v  numTickets=%v",
+			ethcommon.BytesToAddress(sess.OrchestratorInfo.GetTicketParams().GetRecipient()),
+			eth.FormatUnits(batch.FaceValue, "ETH"),
```

Comment:
If we're using this helper here we should probably use it on the [orchestrator's side](https://github.com/livepeer/go-livepeer/blob/83df0883fd82bec79dae54ef5cd86ab7af0aba10/core/orchestrator.go#L178) as well for consistency